chart/kubeapps/README.md

```diff
@@ -702,31 +702,32 @@ Have a look at the [dashboard documentation](https://github.com/kubeapps/kubeapp
 The example below will match the URL `http://example.com` to the Kubeapps dashboard. For further configuration, please refer to your specific Ingress configuration docs (e.g., [NGINX](https://github.com/kubernetes/ingress-nginx) or [HAProxy](https://github.com/haproxytech/kubernetes-ingress)).
 
 ```bash
-helm install kubeapps --namespace kubeapps \
+helm install kubeapps bitnami/kubeapps \
+  --namespace kubeapps \
   --set ingress.enabled=true \
   --set ingress.hostname=example.com \
-    bitnami/kubeapps
+  --set ingress.annotations."kubernetes\.io/ingress\.class"=nginx # or your preferred ingress controller
```

Comment:
We should be using `ingressClassName`, but let's see what the Bitnami chart people think: 
https://github.com/bitnami/charts/pull/7434#issuecomment-915065449